File: threedollar-domain/src/main/java/com/depromeet/threedollar/domain/boss/domain/account/BossAccountRepository.java
package com.depromeet.threedollar.domain.boss.domain.account;

import com.depromeet.threedollar.domain.boss.domain.account.repository.BossAccountRepositoryCustom;
import org.springframework.data.jpa.repository.JpaRepository;

public interface BossAccountRepository extends JpaRepository<BossAccount, Long>, BossAccountRepositoryCustom {

}

File: threedollar-domain-mongo/src/main/kotlin/com/depromeet/threedollar/domain/mongo/boss/domain/account/BossAccountRepository.kt
package com.depromeet.threedollar.domain.mongo.boss.domain.account

import com.depromeet.threedollar.domain.mongo.boss.domain.account.repository.BossAccountRepositoryCustom
import com.depromeet.threedollar.domain.mongo.boss.domain.account.repository.BossAccountStatisticsRepositoryCustom
import org.springframework.data.mongodb.repository.MongoRepository

interface BossAccountRepository : MongoRepository<BossAccount, String>, BossAccountRepositoryCustom, BossAccountStatisticsRepositoryCustom
